src/components/Modal/styles.ts

```diff
@@ -45,7 +45,11 @@ export const Content = styled(Box)<ContentProps>`
         : backgroundColor
       : colors.backgroundLight};
   position: relative;
-  border-radius: 4px;
+  border-radius: ${({
+    theme: {
+      button: { borderRadius },
+    },
+  }: ThemeProp) => borderRadius};
```

Comment:
Yes 

The new CRUK buttons are square to the modal was to have no border radius so the assumption was to instead of having all other brands square to honour the radius that the buttons use

Reply:
stuck to square modals for now